src/components/MonitorLog/ListedSpeciesTab.jsx

```diff
@@ -41,33 +47,94 @@ import {
 import update from 'immutability-helper';
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
-import { useFormContext } from 'react-hook-form';
-import { FiEdit3 } from 'react-icons/fi';
+import { useFormContext, useForm } from 'react-hook-form';
+import { FiEdit3, FiExternalLink } from 'react-icons/fi';
+import { Link } from 'react-router-dom';
 import ListedSpeciesPopup from '../ListedSpecies/ListedSpeciesPopup';
+import GeneralListedInformation from '../ListedSpecies/GeneralListedInformation';
+import Location from '../ListedSpecies/Location';
+import SexSection from '../ListedSpecies/SexSection';
+import BandingSection from '../ListedSpecies/BandingSection';
+import BehaviorsSection from '../ListedSpecies/BehaviorsSection';
+import options from '../ListedSpecies/DropdownOptions';
+import CollapsibleSection from '../CollapsibleSection/CollapsibleSection';
+import NewQuestionModal from '../NewQuestionModal';
+import { OCHBackend } from '../../common/utils';
+
+const ListedSpeciesTab = ({
+  tab,
+  speciesName,
+  speciesCode,
+  speciesId,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'speciesId' is defined but never used.